component/src/test/java/org/wso2/extension/siddhi/io/websocket/sink/util/WebSocketClientConnectorListener.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
A comma missing in the license header. `Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.`

Reply:
fixed in 9fea7d3